src/library/themes/createTheme.js

```diff
@@ -28,119 +26,115 @@ const spacing_3x = pxToEm(24);
-    borderColor_warning_hover: color.orange_70,
-    borderColor_warning_muted: color.orange_40,
+    borderColor_danger_hover: color.red_50,
+    borderColor_success: color.green_60,
+    borderColor_success_active: color.green_70,
+    borderColor_success_focus: color.green_70,
+    borderColor_success_hover: color.green_50,
+    borderColor_warning: color.bronze_60,
+    borderColor_warning_active: color.bronze_70,
+    borderColor_warning_focus: color.bronze_70,
+    borderColor_warning_hover: color.bronze_50,
+
+    boxShadow_1: '0 1px 2px 0 rgba(0,0,0,0.2), 0 2px 4px 0 rgba(0,0,0,0.2)',
+    boxShadow_2: '0 2px 4px 0 rgba(0,0,0,0.2), 0 4px 8px 0 rgba(0,0,0,0.2)',
+    boxShadow_3: '0 4px 8px 0 rgba(0,0,0,0.2), 0 8px 16px 0 rgba(0,0,0,0.2)',
+    boxShadow_4: '0 8px 16px 0 rgba(0,0,0,0.2), 0 20px 16px -8px rgba(0,0,0,0.2)',
+    boxShadow_5: '0 16px 24px 0 rgba(0,0,0,0.2), 0 32px 24px -16px rgba(0,0,0,0.2)',
+    boxShadow_focusInner: color.white,
 
     borderRadius_1: pxToEm(3),
 
     breakpoint_narrow: 512,
     breakpoint_medium: 768,
     breakpoint_wide: 1024,
 
-    color_black: color.black,
-
-    color_caption: color.gray_80,
-
-    ...grays,
-
-    color_h1: color.gray_100,
-    color_h2: color.gray_80,
-    color_h3: color.gray_80,
-    color_h4: color.gray_80,
-    color_h5: color.gray_100,
-    color_h6: color.gray_80,
+    color_disabled: color.gray_50,
+    color_danger: color.red_60,
+    color_danger_active: color.red_70,
+    color_danger_focus: color.red_60,
+    color_danger_hover: color.red_50,
     color_mouse: color.gray_80,
-    color_placeholder: color.gray_60,
-
-    ...primaries,
-
+    color_primary: color.white,
+    color_readOnly: color.gray_80,
+    color_required: color.red_60,
+    color_success: color.green_60,
+    color_success_active: color.green_70,
+    color_success_focus: color.green_60,
+    color_success_hover: color.green_50,
     color_text: color.gray_100,
-    color_text_disabled: color.gray_50,
-    color_text_danger: color.red_60,
-    color_text_danger_active: color.red_70,
-    color_text_danger_focus: color.red_60,
-    color_text_danger_hover: color.red_50,
-    color_text_ondanger: color.white,
-    color_text_primary: primaries.color_theme_60,
-    color_text_primary_active: primaries.color_theme_70,
-    color_text_primary_focus: primaries.color_theme_60,
-    color_text_primary_hover: primaries.color_theme_50,
-    color_text_onprimary: color.white,
-    color_text_success: color.green_90,
-    color_text_success_active: color.green_100,
-    color_text_success_focus: color.green_90,
-    color_text_success_hover: color.green_80,
-    color_text_onsuccess: color.white,
-    color_text_warning: color.orange_70,
-    color_text_warning_active: color.orange_80,
-    color_text_warning_focus: color.orange_70,
-    color_text_warning_hover: color.orange_60,
-    color_text_onwarning: color.white,
-
-    color_white: color.white,
+    color_text_inverted: color.white,
```

Comment:
this doesn't seem to fit the [target]_[property]_[variant]_[state] schema. or am I missing something?

Reply:
You're right. I made `color_text` and `color_text_inverted` rather than the more appropriate `color` and `color_inverted` to avoid the potentially-confusing `color` theme key, but that concern isn't outweighed by the inconsistency.